db/migrate/20221102182759_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CreatePosts < ActiveRecord::Migration[7.0]
+    def change
+      create_table :posts do |t|
+        t.references :author, index: true, foreign_key: { to_table: :users }
+        t.string :title
+        t.string :text
```

Comment:
- [ ] Kindly, instead of using `string` data type for the text column, I'll suggest you use `text` data type. string field has a limited number of characters(255 approximatively) while the text field could allow you to write up 30000 characters.